packages/store/src/secure-store/keytar-store.ts

```diff
@@ -53,7 +53,9 @@ export class KeytarStore implements SecureStore {
         try {
             return (await this.keytar.findCredentials(service)).reduce((result, entry) => {
                 try {
-                    result[String(entry.account)] = JSON.parse(entry.password);
+                    if (entry.account) {
```

Comment:
Could you please add a comment about when a user denies permission to access the keystore that the `entry.account` is undefined (if that is whats happening?), thx

Reply:
Thank you, added the comment.